kmq.js

```diff
@@ -10,13 +11,15 @@ const guessSong = require("./helpers/guess_song");
 const validate = require("./helpers/validate");
 const options = require("./commands/options");
 const { clearPartiallyCachedSongs } = require("./helpers/utils");
+
 let db;
 let commands = {};
 let gameSessions = {};
 let guildPreferences = {};
 
 client.on("ready", () => {
     logger.info(`Logged in as ${client.user.tag}!`);
+    postTopGGCount();
```

Comment:
This needs to happen outside of `.on("ready")` because the top.gg API waits for the same event